parsing_scripts/twitch_parser.py

```diff
@@ -1,8 +1,17 @@
 from parsing_scripts.base_parser import AbstractParser
+from dao.container_kafka import Kafka
+from dao.container_mongo import ContainerMongo
+import asyncio
 
 
 class TwitchParser(AbstractParser):
     BASE_URL = "https://api.twitch.tv/helix/streams"
 
-    async def parse(self):
+    async def parse(self, k: Kafka, mongo: ContainerMongo):
         soup = await self.get_soup(self.BASE_URL)
+        for i in range(10):
+            d = k.producer.send('topic_twitch', value={'counter': 'twitch'})
+            data = d.get()
+            print(f'send {data}')
+            await asyncio.sleep(1)
```

Comment:
you can provide Kafka class inside __init__, or configure() method for instance and remove it from parse method signature, so you would rather do self.kafka.producer.send(...)

Reply:
Done with init def